pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -96,16 +104,40 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 	if o.Client, err = kubernetes.NewForConfig(o.Config); err != nil {
 		return err
 	}
+	if o.ImageClient, err = imageclient.NewForConfig(o.Config); err != nil {
+		return err
+	}
+	return err
+}
+
+func (o *MustGatherOptions) Complete(cmd *cobra.Command, args []string) error {
+	var err error
+	if i := cmd.ArgsLenAtDash(); i != -1 && i < len(args) {
+		o.Command = args[i:]
+	} else {
+		o.Command = args
+	}
 	if len(o.DestDir) == 0 {
 		o.DestDir = fmt.Sprintf("must-gather.local.%06d", rand.Int63())
 	}
-	if len(o.Image) == 0 {
-		if o.Image, err = o.resolveMustGatherImage(); err != nil {
-			o.Image = "quay.io/openshift/origin-must-gather:latest"
-			fmt.Fprintf(o.Out, "%v\n", err)
+	for _, imageStream := range o.ImageStreams {
+		if image, err := o.resolveImageStreamTagString(imageStream); err == nil {
+			o.Images = append(o.Images, image)
+		} else {
+			o.log("Unable to resolve image stream: %v", err)
 		}
 	}
-	fmt.Fprintf(o.Out, "Using image: %s\n", o.Image)
+	if len(o.Images) == 0 {
+		var image string
+		if image, err = o.resolveImageStreamTag("openshift", "must-gather", "latest"); err != nil {
+			o.log("%v\n", err)
+			image = "quay.io/openshift/origin-must-gather:latest"
+		}
+		o.Images = append(o.Images, image)
+	}
+	for _, image := range o.Images {
+		o.log("Using must-gather plugin-in image: %s", image)
+	}
```

Comment:
Why not just:
```go
o.log("Using must-gather plugin-in images: %s", strings.Join(o.Images, ", "))
```
instead of this entire for loop?

Reply:
Done